pkg/cli/cmd/radinit/environment.go

```diff
@@ -96,8 +96,12 @@ func (r *Runner) CreateEnvironment(ctx context.Context) error {
 	}
 
 	if r.Options.CloudProviders.Azure != nil {
-		credential := r.getAzureCredential()
-		err := credentialClient.PutAzure(ctx, credential)
+		credential, err := r.getAzureCredential()
+		if err != nil {
+			return clierrors.MessageWithCause(err, "Failed to configure Azure credentials.")
```

Comment:
Should this be `Failed to get Azure credentials`? It looks like it is the same as the error message below (after **PutAzure**).

Reply:
`getAzureCredential` just returns the `AzureCredentialResource` so I think the error message with the error attached it more correct than `Failed to get Azure credentials`.

```go
func (r *Runner) getAzureCredential() (ucp.AzureCredentialResource, error) {
	switch r.Options.CloudProviders.Azure.CredentialKind {
	case azure.AzureCredentialKindServicePrincipal:
		return ucp.AzureCredentialResource{
			Location: to.Ptr(v1.LocationGlobal),
			Type:     to.Ptr(cli_credential.AzureCredential),
			Properties: &ucp.AzureServicePrincipalProperties{
				Storage: &ucp.CredentialStorageProperties{
					Kind: to.Ptr(ucp.CredentialStorageKindInternal),
				},
				TenantID:     &r.Options.CloudProviders.Azure.ServicePrincipal.TenantID,
				ClientID:     &r.Options.CloudProviders.Azure.ServicePrincipal.ClientID,
				ClientSecret: &r.Options.CloudProviders.Azure.ServicePrincipal.ClientSecret,
			},
		}, nil
	case azure.AzureCredentialKindWorkloadIdentity:
		return ucp.AzureCredentialResource{
			Location: to.Ptr(v1.LocationGlobal),
			Type:     to.Ptr(cli_credential.AzureCredential),
			Properties: &ucp.AzureWorkloadIdentityProperties{
				Storage: &ucp.CredentialStorageProperties{
					Kind: to.Ptr(ucp.CredentialStorageKindInternal),
				},
				TenantID: &r.Options.CloudProviders.Azure.WorkloadIdentity.TenantID,
				ClientID: &r.Options.CloudProviders.Azure.WorkloadIdentity.ClientID,
			},
		}, nil
	default:
		return ucp.AzureCredentialResource{}, fmt.Errorf("unsupported Azure credential kind: %s", r.Options.CloudProviders.Azure.CredentialKind)
	}
}
```